virttest/qemu_vm.py

```diff
@@ -104,14 +104,20 @@ def qemu_proc_term_handler(vm, monitor_exit_status, exit_status):
     Callback function to detect QEMU process non-zero exit status and
     push VMExitStatusError to background error bus.
 
-    :param vm: VM name.
+    :param vm: VM object.
     :param monitor_exit_status: True to push VMUnexpectedExitError instance
         with calltrace to global error event bus.
     :param exit_status: QEMU process exit status.
     """
+    for img_sn in vm.devices.temporary_image_snapshots:
```

Comment:
pop items in the list looks more pretty.

Reply:
Thanks, updated.